app/controllers/map.ts

```diff
@@ -78,8 +78,7 @@ export default class MapComponent extends Controller {
         this.agendaData.forEach((agendaItem: AgendaItemModel) => {
           const agendaItemLocation = agendaItem
             .get('session')
-            ?.get('governingBody')
-            ?.get('administrativeUnit')?.name;
+            ?.get('municipality');
```

Comment:
`.get` is usually a code smell and means that the async relationship isn't handled correctly. I think it might not be needed here anymore if the session getter is updated.